brave-core/src/main/java/com/github/kristofa/brave/BraveContext.java

```diff
@@ -0,0 +1,68 @@
+package com.github.kristofa.brave;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+/**
+ * The {@link BraveContext} is a registry of all {@link Brave} instances that have been created.
```

Comment:
is that this is a registry of all instances intentional, or rather a reality of the implementation.

Ex. I understand that if this registers on each new Brave, that this could have many Braves.. wondering if there is a use case for iterating through Braves, or if the use case is just "give me a brave"